docs/source/models/concept_models/vivarium_mncnh_portfolio/facility_model.rst

```diff
@@ -0,0 +1,106 @@
+    copula = GaussianCopula([[1., .5, .1],
+                             [.5, 1., .1],
+                             [.1, .1, 1.]])
+    copula.rvs(10)
+
+.. list-table:: Propensity Correlation Stand-in Parameters
+   :header-rows: 1
+   :widths: 20 20 20
+
+   * - Factor A
+     - Factor B
+     - Correlation
+   * - ANC Propensity
+     - In-facility Birth Propensity
+     - 0.50
+   * - ANC Propensity
+     - LBWSG Propensity
+     - 0.10
+   * - In-facility Birth Propensity
+     - LBWSG Propensity
+     - 0.10
+
+Eventually we must put draws of consistent values in the artifact.
+
+Special ordering of the categories
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our method of inducing correlations using a Gaussian copula is equivalent to specifying the `polychoric correlation <https://en.wikipedia.org/wiki/Polychoric_correlation>`_ between the variables, and relies on having a known ordering of variable values.
+
+When we calibrate the model, we will use an ordering of the LBWSG categories that makes them most likely to correlate with the ANC and home delivery propensities.  To achieve this, we will order them from lowest average RR to highest average RR (averaged across all draws).  Note that this is sex-specific, and based on the RRs for the early neonatal age group!  This ordering must also be used in the Risk component.
+
+We will also order the ANC and home delivery propensities from lowest to highest risk: some ANC < no ANC; and CEmONC < BEmONC < home birth.
+
+
+[[Perhaps more details to come, such as code or an explicit table.]]
+
+Conditional probabilities of home delivery
+------------------------------------------
+
+In addition to correlation, we posit that a belief about preterm status is influential in the decision to have a home delivery.  We will model this as a conditional probability of home delivery given a belief about preterm status.  Although deriving consistent values for these conditional probabilities is complex, and described in the final section of this page, *using* the conditional probabilities is simple: simply select the facility type with :math:`\text{Pr}[\text{facility}\mid\text{believed preterm}]` and :math:`\text{Pr}[\text{facility}\mid\text{believed fullterm}]` for the corresponding cases.
+
+.. list-table:: Conditional Probability Stand-in Parameters
+   :header-rows: 1
+   :widths: 20 20 20 20
+
+   * - Belief
+     - Home
+     - BeMONC
+     - CeMONC
+   * - Term
+     - 0.25
+     - 0.10
+     - 0.65
+   * - Preterm
+     - 0.05
+     - 0.05
+     - 0.90
+
+Challenge of calibrating the model
+----------------------------------
+
+We have developed a nonlinear optimization model to find a consistent set of parameters for the Gaussian copula and the conditional probabilities.
+It will be described in detail here.
+
+Link to code implementing it, too.
+
+
+Range of propensity and probabilities (and probability gaps) that are consistent with existing data
```

Comment:
What do you mean by "probability gaps"?

Reply:
I was thinking of the difference between the conditional probabilities conditioned on believed full term and believed preterm. I'll add some text to make it clearer.